tools/docker/Dockerfile.test_spack

```diff
@@ -41,15 +42,27 @@ RUN python3 -m venv /opt/venv
 ENV PATH="/opt/venv/bin:$PATH"
 RUN pip3 install --quiet boto3==1.38.11 google-cloud-storage==3.1.0
 
-# Install a recent Spack version
+# Install Spack and Spack packages
 WORKDIR /root/spack
 ARG SPACK_VERSION
-ENV SPACK_VERSION=${SPACK_VERSION:-develop-2025-05-18}
-RUN git init --quiet && \
-    git remote add origin https://github.com/spack/spack.git && \
-    git fetch --quiet --depth 1 origin ${SPACK_VERSION} --no-tags && \
-    git checkout --quiet FETCH_HEAD
-ENV PATH="/root/spack/bin:${PATH}"
+ENV SPACK_VERSION=${SPACK_VERSION:-2e7168b4cfe9222a94becf0c2c5b401f513181ec}
+ARG SPACK_PACKAGES_VERSION
+ENV SPACK_PACKAGES_VERSION=${SPACK_PACKAGES_VERSION:-ec6c0f3435d3a862337a7af71d07790f4d116274}
+ARG SPACK_REPO=https://github.com/spack/spack
+ENV SPACK_ROOT=/opt/spack-$SPACK_VERSION
+ARG SPACK_PACKAGES_REPO=https://github.com/spack/spack-packages
+ENV SPACK_PACKAGES_ROOT=/opt/spack-packages-$SPACK_PACKAGES_VERSION
+RUN mkdir -p $SPACK_ROOT \
+    && curl -OL $SPACK_REPO/archive/$SPACK_VERSION.tar.gz \
+    && tar -xzvf $SPACK_VERSION.tar.gz -C /opt && rm -f $SPACK_VERSION.tar.gz \
```

Comment:
I suggest to drop the `v` flag for the `tar` commands (see also below) to reduce the amount of output.